Resources/Prototypes/_NF/Entities/Effects/portal.yml

```diff
@@ -0,0 +1,16 @@
+- type: entity
+  id: BasePortalAdmin
+  perent: BasePortal
```

Comment:
Typo here, should be `parent`.  Hotfix inbound?

Reply:
Go for it